api/envoy/config/trace/v2/trace.proto

```diff
@@ -196,14 +196,18 @@ message OpenCensusConfig {
   // The URL to Zipkin, e.g. "http://127.0.0.1:9411/api/v2/spans"
   string zipkin_url = 6;
 
-  // Enables the OpenCensus Agent exporter if set to true. The address must also
+  // Enables the OpenCensus Agent exporter if set to true. The address or grpc service must also
   // be set.
   bool ocagent_exporter_enabled = 11;
 
   // The address of the OpenCensus Agent, if its exporter is enabled, in gRPC
   // format: https://github.com/grpc/grpc/blob/master/doc/naming.md
   string ocagent_address = 12;
 
+  // (optional) The gRPC server hosted by the OpenCensus Agent. Only Google gRPC is supported.
+  // This is only used if the address is left empty.
```

Comment:
Hum, seems like this is a good case for `oneof`? We can deprecate `ocagent_address` and have a new field that is `oneof` string or `api.v2.core.GrpcService`. @lizan WDYT?

Reply:
I was following the terminology from line 199 above, I can update.

I was thinking it might make sense to consolidate with field 13 which was just added for stackdriver, since the x_exporter_enabled field would differentiate which exporter the grpc service should be used for, however I'm unsure of the policy for changing around protos so just went with something that was strictly adding a field instead of changing fields around.

Reply:
Ideally we can just deprecate `ocagent_address`, but for current situation we can't deprecate any field until we boost v4alpha, can you leave a `[#comment:TODO: deprecate this field]` for `ocagent_address`?